src/Solver/Deterministic/DeDriver.cpp

```diff
@@ -121,6 +124,70 @@ DSP_RTN_CODE DeDriver::run()
 			osi_->si_->setInteger(j);
 	}
 
+	/** add quadratic constraints */
+	if (model_->isQuadratic())
+	{
+		DecTssQcModel * qcModel;
+		try
+		{
+			qcModel = dynamic_cast<DecTssQcModel *>(model_);
+		}
+		catch (const std::bad_cast& e)
+		{
+			printf("Model claims to be quadratic when it is not");
+			return DSP_RTN_ERR;
+		}
+
+		for (int s = 0; s < qcModel->getNumScenarios(); s++) {
+			
+			QcRowDataScen * qcrowdata = qcModel->getQRowsCPXParams(s);
```

Comment:
Is this CPLEX-specific?

Reply:
No, changed the name to eliminate confusion.